lib/dataTrigger/DataTriggerExecutionContext.js

```diff
@@ -0,0 +1,17 @@
+class DataTriggerExecutionContext {
+  /**
+   * @param {DataProvider} dataProvider
+   * @param {string} userId
+   * @param {Contract} contract
+   */
+  constructor(dataProvider, userId, contract) {
```

Comment:
I think we should pass ST as well. We have it in verification function where we are going to call triggers.

Reply:
What do you mean? There is no ST in any of the packet verification functions

Reply:
https://github.com/dashevo/js-dpp/blob/c8eefeac2435b7219ab8af1362d7cf84afb81ec6/lib/stPacket/verification/verifySTPacketFactory.js#L23

Reply:
Yeah, but that a packet, not an ST

Reply:
Done